cmd/exporter/exporter.go

```diff
@@ -92,6 +90,22 @@ func healthProbe(w http.ResponseWriter, req *http.Request) {
 	}
 }
 
+func finalizing() {
```

Comment:
I must;ve missed this in the rebase... 
I do need the finalizing to do the accelerator clean up... let me see if I can work around this